app-center-services/src/main/java/org/exoplatform/appcenter/service/ApplicationCenterService.java

```diff
@@ -814,5 +843,12 @@ private Long getDefaultImageId() {
 
     return userApplicationsList;
   }
+  public static void broadcastEvent(ListenerService listenerService, String eventName, Object source, Object data) {
```

Comment:
Is there a need to mark this class as static ?
besides listenerService is an attribute of this class and you can use it directly